server/app/api/registry/registry_controller.py

```diff
@@ -10,7 +11,7 @@
 def registry_endpoint(
     request: Request, 
     registry_service: RegistryService = Depends(get_registry_service),
-    scope: str | None = None,
+    scopes: List[str] | None = Query(default=None, alias="scope"),
     service: str | None = None):
```

Comment:
I like that FastAPI supports list query parameters so elegantly. The usage of `alias` here is also really clever.